src/adserver.js

```diff
@@ -0,0 +1,57 @@
+import {formatQS} from './url';
```

Comment:
After discussion we decided to keep it adserver.js as of now. When more adservers are added it will be updated